example/tags/tags.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
example tags provided by default for openmct. these should be modified by admins for individual installations. we may want to move this to a different directory. also, if the file is empty should we disable tagging?